hw2/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { User } from './models/user.model';
+
+let usersList: User[] = [
+    {
+        id: '1',
+        login: 'Abc',
+        password: '123',
+        age: 1,
+        isDeleted: false
+    },
+    {
+        id: '2',
+        login: 'Bc',
```

Comment:
You define login in the Joi schema as email. Stick with it in seed data, to make the app consistent.